pkg/controller/controller.go

```diff
@@ -101,6 +101,11 @@ const (
 	nodePublishSecretNameKey      = "csiNodePublishSecretName"
 	nodePublishSecretNamespaceKey = "csiNodePublishSecretNamespace"
 
+	// PV and PVC metadata, used for sending to drivers in the  create requests, added as parameters, optional.
+	pvcNameKey      = "kubernetes.io/created-for/pvc/name"
+	pvcNamespaceKey = "kubernetes.io/created-for/pvc/namespace"
+	pvNameKey       = "kubernetes.io/created-for/pv/name"
```

Comment:
cc @saad-ali is this an acceptable prefix? maybe it should be one of the volume specific prefixes?

Reply:
I would prefer to use the same prefix as for [pod info on mount](https://kubernetes-csi.github.io/docs/pod-info.html): `csi.storage.k8s.io/...`

Reply:
Sounds good, updated.